### PR TITLE
fix types

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: src/steps/fetch-teams/index.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ const convertTeams = (teams: Teams): Entity[] =>
 const step: IntegrationStep = {
   id: 'fetch-org-teams',
   name: 'Fetch Organization Teams',
-  types: ['npm_team'],
+  types: ['npm_team', 'npm_team_has_user'],
```
Comment (erkangz) [inline src/steps/fetch-teams/index.ts:32]: @aiwilliams @charlieduong94 something to think about, it's not clear to the developer what is the relationship type produced by the `createIntegrationRelationship` function, therefore difficult to know what needs to be put into here

File: src/steps/fetch-packages/index.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ const convertPackages = (packages: Packages): Entity[] =>
 const fetchPackages: IntegrationStep = {
   id: 'fetch-org-packages',
   name: 'Fetch Organization Packages',
-  types: ['npm_package'],
+  types: ['npm_package', 'repo_published_npm_package'],
```
Comment (erkangz) [inline src/steps/fetch-packages/index.ts:49]: @aiwilliams @philidem @charlieduong94 should we include mapped relationships here?

Comment (aiwilliams) [reply src/steps/fetch-packages/index.ts:49]: An integration is declaring the complete, new set of relationships - including mapped - it expects to exist after synchronization, and therefore, should it fail to produce that complete set, the synchronizer needs to know that the complete set is unknown. This `types` mechanism produces a "partial set" indicator for the synchronizer. Without including the `_type` of the mapped relationships, it cannot know to NOT delete those relationships when the set is incomplete.

File: src/steps/fetch-teams/index.ts (hunk 1 above)
Comment (aiwilliams) [reply src/steps/fetch-teams/index.ts:32]: I agree, I ran into this on the Duo integration. What I've been doing in the Azure integration is something like this:
```
COSMOSDB_ACCOUNT_TYPE = "azure_cosmosdb_account";
COSMOSDB_SQL_DB_TYPE = "azure_cosmosdb_sql_database";
ACCOUNT_COSMOSDB_SQL_DB_RELATIONSHIP_TYPE = generateRelationshipType(
   DataModel.RelationshipClass.HAS,
   COSMOSDB_ACCOUNT_TYPE,
   COSMOSDB_SQL_DB_TYPE,
)
```
The `generateRelationshipType` is used by the `createIntegrationRelationship` function.

This is a lot of fanfare. Perhaps folks should just write out the type name, and we have a validator that looks at the from/to entity `_type` values and checks to see if they followed the naming pattern. I really don't much care for the constants at all, so using `generateRelationshipType` isn't too much worse IMO. 😆

**Pull request merged**